consensus/spos/commonSubround/subroundStartRound.go

```diff
@@ -163,9 +172,18 @@ func (sr *SubroundStartRound) initCurrentRound() bool {
 	sr.SetStatus(sr.Current(), spos.SsFinished)
 
 	if leader == sr.SelfPubKey() {
+		if sr.appStatusHandler != nil {
```

Comment:
should move the check in SetAppStatusHandler func

Reply:
done